### DISCUSSION TITLE
Aktuelle Binärdateien

### DISCUSSION THREAD

**Discussion by ChaosChris1975** [tbnobody/OpenDTU#295] [Q&A]
Hallo !
So ganz durchschaue ich das mit Github noch nicht.
Ist die aktuelle ZIP-Datei mit den Binärfiles  die mit der höchsten "#"Nummer bei Actions / OpenDTU Build ??
Wäre es nicht irgendwie möglich die aktuelle auf der Startseite wo die Readme ist, zu verlinken ? Oder finde ich das nur nicht ?

**Comment by tbnobody** [comment]
> Ist die aktuelle ZIP-Datei mit den Binärfiles die mit der höchsten "#"Nummer bei Actions / OpenDTU Build ??

Die Liste ist absteigend nach Datum sortiert. Das erste Element (mit dem Branch 'master') ist immer das aktuelle.

**Reply by ChaosChris1975** [reply]
Danke, hab jetzt auch über der Readme den Link gefunden.

Muss dir hier mal ein dickes Lob aussprechen, einfach ein tolles Projekt.
